src/base/projects/Header.svelte

```diff
@@ -21,8 +22,14 @@
   export let branches: [string, string][] = [];
   export let content: ProjectContent;
   export let revision: string;
+  // If parent component is a seed page
+  export let isSeedPage: boolean;
```

Comment:
We shouldn't imply knowledge of the parent's logic, this should be called something like `peerSelector`.